### PR TITLE
Intersection of LinkedLists in Python #1247

### LINKED ISSUE #1247: Finding interesection node of two linked lists connected in y shape
## 🚀 Feature

There are two singly linked lists in a system. The end node of one of the linked list got linked to the second list, forming an inverted Y-shaped list. Write a program to get the point where two linked lists merge in Python 

## Motivation

Linked Lists are one of the data structures to which basic to advanced coding interview questions are related. Hence as many as possible algorithms should be learned on linked lists.

## Pitch
Linked lists are a kind of data structure which have numerous applications. Many solutions can be made using linked lists as a data structure. Hence learning as many as possible algorithms related to it helps us resolve many issues in programming.

## Self-assign 
 I would request you to approve this issue and assign it to me under the label"GSSoC'21" and "Level-2" or "Level-3"
Thank you

### PR DESCRIPTION
**Pull request by Nikitha2309**
There are two singly linked lists in a system. The end node of one of the linked list got linked to the second list, forming an inverted Y-shaped list. Write a program to get the point where two linked lists merge in Python

Fixes #1247

### CODE REVIEW

**Comment by Nikitha2309** [comment]
@raksha009 
Pls review this PR 
Thank You

**Comment by Nikitha2309** [comment]
@plazzy99 @raksha009 
Pls review this PR
Thank you

**Comment by Nikitha2309** [comment]
Respected @tarun26091999 
Please assign this issue to some other else and make it reviewed
I have written a clean code with comments and made all necessary changes.

It has been so long since I posted this PR , but no one reviewed this yet
Bcoz of this I am getting Branch conflicts
So Please REVIEW THIS PR 
Thank you

File: Code/Python/linked_lists_intersection.py [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+Output Format :
+
+Example : ( Output to the above input example ) 
+
+Linked List 1 = 1->2->3->4->5->6->7->None
+Linked List 2 = 5->4->3->4->5->6->7->None
+The intersection point is 3
+
+Corner Test Cases Result
+Code works fine for all corner test cases tested , including null linked lists.
```
Comment (tarun26091999) [inline Code/Python/linked_lists_intersection.py:158]: remove these lines

**Review by tarun26091999 — CHANGES_REQUESTED** [review]
add one more testcase.

File: Code/Python/linked_lists_intersection.py (hunk 1 above)
Comment (Nikitha2309) [reply Code/Python/linked_lists_intersection.py]: Done

**Comment by Nikitha2309** [comment]
Requested Changes Done @tarun26091999 
Let me know if there is anything else to be done
Merge if there are none
Thank you

**Review by rudrakshi99 — APPROVED** [review]

**Pull request merged**